data/spec_adopters.yml

```diff
@@ -1,6 +1,10 @@
 ---
 adopters:
   projects:
+  - name: Madras JUG
+    homepage_url: https://www.meetup.com/MadrasJUG/
+    logo: mjug.svg
+    logo_white: mjug.svg
   
 # Template: Example for creating new specification entry
 #  - name: Jakarta Bean Validation
```

Comment:
Sorry, I missed a part. This is how the file should look like:

```
---
adopters:
  projects:
  - name: Jakarta JSON Processing
    directory: specifications/jsonp/ 
    adopters:
    - name: Madras JUG
      homepage_url: https://www.meetup.com/MadrasJUG/
      logo: mjug.svg
      logo_white: mjug.svg
```
You will also need to sign the ECA with the email you are using for your GitHub account. This should be the same as for the Eclipse Foundation account. Log in to https://accounts.eclipse.org/ and check the status of your ECA in the right hand menu.